integration_tests/tests/test_lambda_functions.py

```diff
@@ -24,16 +27,21 @@ def lambda_execution_timestream_metrics_summary(region, start_epochtimemsec, sta
     query = f"""SELECT count(*) as executions
                 FROM "{DB_NAME}"."{TABLE_NAME}"
                 WHERE measure_name = 'execution' AND time > from_milliseconds({start_epochtimemsec})
-    """   
+    """
     result = query_runner.execute_query(query=query)
 
-    return result[0] # returning the first record (it's only 1 record in resultset by query design)
+    return result[
+        0
+    ]  # returning the first record (it's only 1 record in resultset by query design)
```

Comment:
if we put comment on a line before (not the same one), formatting would be less weird :)

Reply:
noted, thank you :)